healthcare/api-client/v1/fhir/fhir_resources_test.py

```diff
@@ -0,0 +1,304 @@
+# Copyright 2018 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pytest
+import sys
+import uuid
+
+from googleapiclient.errors import HttpError
+from retrying import retry
+
+# Add datasets for bootstrapping datasets for testing
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", "datasets"))  # noqa
+import datasets
+import fhir_stores
+import fhir_resources
+
+cloud_region = "us-central1"
+project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
+service_account_json = os.environ["GOOGLE_APPLICATION_CREDENTIALS"]
+
+bundle = os.path.join(os.path.dirname(__file__), "resources/execute_bundle.json")
+dataset_id = "test_dataset_{}".format(uuid.uuid4())
+fhir_store_id = "test_fhir_store-{}".format(uuid.uuid4())
+resource_type = "Patient"
+
+
+def retry_if_server_exception(exception):
+    return isinstance(exception, (HttpError))
+
+
+@pytest.fixture(scope="module")
+def test_dataset():
+    @retry(
```

Comment:
FYI, I replaced `retrying` with `backoff` in https://github.com/GoogleCloudPlatform/python-docs-samples/pull/3434.

It's up to you to follow.

Reply:
Done.